src/components/ArmyList.vue

```diff
@@ -19,39 +19,74 @@ const points = computed(() => {
   );
 });
 
+
+function sortName(a, b) {
+  const aa = a.name.toLowerCase();
+  const bb = b.name.toLowerCase();
+  return aa < bb ? -1 : aa > bb ? 1 : 0;
+}
+
+function sortPointsMin(a, b) {
+  return a.points < b.points ? -1 : a.points > b.points ? 1 : 0;
+}
+
+function sortPointsMax(a, b) {
+  return a.points < b.points ? 1 : a.points > b.points ? -1 : 0;
+}
+
 const emptySpace = computed(() => {
   return (
     Math.max(0, props.appData.currentList.maxPoints - points.value) *
-      scale.value +
+    scale.value +
     "px"
   );
 });
 </script>
 
 <template>
-  <draggable
-    v-model="props.appData.currentList.units"
-    group="units"
-    animation="150"
-    item-key="id"
-    class="army-list"
-  >
-    <template #item="{ element, index }">
-      <ArmyListUnit :unit="element" :scale="scale" :app-data="props.appData" />
-    </template>
-  </draggable>
+  <div class="sidebar">
+    <div class="sidebar-sort-label">Sort By:</div>
+    <span class="sidebar-sort-box">
+      <button @click="props.appData.currentList.units.sort(sortPointsMax)">Points Max</button>
+      <button @click="props.appData.currentList.units.sort(sortPointsMin)">Points Min</button>
+      <button @click="props.appData.currentList.units.sort(sortName)">Name</button>
+    </span>
```

Comment:
Adding these elements to the `ArmyList` causes the first item in the list to be bumped off the page.
![image](https://github.com/furka/40k-10th-list-builder/assets/2657994/84e6026f-20f1-46a8-bd85-2e53f63ef783)

It will also mess with the calculation for each unit's size based on points cost.
https://github.com/furka/40k-10th-list-builder/blob/master/src/components/ArmyList.vue#L11

Not sure what the best solution for this is

Reply:
Hmm. Lazy of me to miss that. The solution is probably some esoteric CSS.

I'll fix it.

Reply:
Okay. Please try again.